README.md

```diff
@@ -170,9 +170,23 @@ offer to edit the whitelist instead.
 - `[--mysql=PORT_OR_SOCKET]`: Use MySQL/MariaDB port or socket
 
 
+
+If you are using multiple databases per environment, pass the database name like this:
+
+    geordi dump -d <name of key where db is nested>
+
+Ususally in Rails the first database is nested under `primary`.
+
+Loading a dump into one of multiple local databases is not supported yet.
+
+When called with the `--load` option, sources the specified dump into the
+development database.
+
+
```

Comment:
This looks mislocated as well. Please run `rake readme` and check if it fixed this.